src/components/link/link.spec.tsx

```diff
@@ -256,12 +256,12 @@ describe("Link", () => {
 
   describe("onClick prop", () => {
     it("should render a button element", () => {
-      wrapper = renderLink({ onClick: () => {} });
+      wrapper = renderLink({ onClick: () => null });
```

Comment:
shouldn't that be `() => void`?

Reply:
This isn't a type, we're actually setting `onClick` to a function that returns null for the purposes of the test. It could be null or undefined, it doesn't really matter for the test.